RushHour/RushHourMod.cs

```diff
@@ -22,13 +22,17 @@ public void OnSettingsUI(UIHelperBase helper)
                 new OptionsCheckbox() { readableName = "EXPERIMENTAL: Use improved commercial demand", value = false, uniqueName = "UseImprovedCommercial" }
             };
 
+            loadSettingsFromSaveFile();
```

Comment:
I am new to modding Cities, is there a better place for this call to actually live? Lifecycle-wise this is working on load regardless of if I visit the settings page or not but it does not feel like it's in its right semantic place.